site/blocks/RelatedCharts.tsx

```diff
@@ -48,7 +49,15 @@ export const RelatedCharts = ({ charts }: { charts: RelatedChart[] }) => {
                                         }
                                     }}
                                 >
-                                    {chart.title}
+                                    <img
+                                        src={`${BAKED_GRAPHER_EXPORTS_BASE_URL}/${chart.slug}.svg`}
+                                        loading="lazy"
+                                        data-no-lightbox
+                                        data-no-img-formatting
+                                        width={850}
+                                        height={600}
```

Comment:
I wonder if we should extract these into `GrapherConstants.ts` so they are synchronised between site and grapher?

Reply:
yep makes sense, thanks! I've replaced them in a couple more places in grapher code too.